aws-analytics-pinpoint/src/androidTest/java/com/amplifyframework/analytics/pinpoint/AnalyticsPinpointInstrumentedTest.java

```diff
@@ -138,6 +139,66 @@ public void testAutoFlush() throws AnalyticsException {
         assertEquals(0, analyticsClient.getAllEvents().size());
     }
 
+    /**
+     * Registers a global property and ensures that all recorded events have the global property.
+     * @throws AnalyticsException Caused by incorrect usage of the Analytics API.
+     * @throws HubException thrown if an error condition is encountered while registering global properties
+     *                      and Amplify fails to publish the error message to the hub.
+     * @throws JSONException Caused by unexpected event structure.
+     */
+    @Test
+    public void testRegisterGlobalProperties() throws AnalyticsException, HubException, JSONException {
+        // Register a global property
+        registerGobalProperty();
+
+        BasicAnalyticsEvent event = new BasicAnalyticsEvent("Amplify-event" + UUID.randomUUID().toString(),
+                PinpointProperties.builder()
+                        .add("Property", "PropertyValue")
+                        .build());
+        Amplify.Analytics.recordEvent(event);
+        Amplify.Analytics.recordEvent("amplify-test-event");
+
+        JSONObject eventAttributes =
+                new JSONObject(analyticsClient.getAllEvents().get(0).get("attributes").toString());
+        JSONObject event2Attributes =
+                new JSONObject(analyticsClient.getAllEvents().get(1).get("attributes").toString());
+
+        assertEquals(2, analyticsClient.getAllEvents().size());
+        assertEquals(true, eventAttributes.has("Property"));
+        assertEquals(true, eventAttributes.has("GlobalProperty"));
+        assertEquals(false, event2Attributes.has("Property"));
+        assertEquals(true, event2Attributes.has("GlobalProperty"));
+    }
+
+    /**
+     * Registers a global property and then Unregisters the global property
+     * and ensures that it is respected in events recorded thereafter.
+     * @throws HubException thrown if an error condition is encountered while registering global properties
+     *                      and Amplify fails to publish the error message to the hub.
+     */
+    @Test
+    public void testUnregisterGlobalProperties() throws HubException {
+        // Register a global property
+        registerGobalProperty();
+
+        // Unregister global property
+        Set<String> globalPropertyKeys = new HashSet<>();
+        globalPropertyKeys.add("GlobalProperty");
+        Amplify.Analytics.unregisterGlobalProperties(globalPropertyKeys);
+
+        Amplify.Analytics.recordEvent("amplify-test-event-without-property");
+
+        assertEquals(1, analyticsClient.getAllEvents().size());
+        assertEquals(false, analyticsClient.getAllEvents().get(0).has("attributes"));
```

Comment:
`assertEquals(false, something)` can be simplified as `assertFalse(something)`.

Reply:
Done.